frontend/src/hooks/useKeycloakWrapper.tsx

```diff
@@ -51,6 +51,7 @@ function useKeycloakWrapper(): IKeycloak {
    * @param claim - The name of the claim
    */
   const hasClaim = (claim?: string | Array<string>): boolean => {
+    userInfo.roles = keycloak?.realmAccess?.roles || [];
```

Comment:
Hmm... what is the motivation for this? I think userInfo should contain roles already (via a mapping in keycloak). Am I correct @Fosol ?

Reply:
My bad, I was only checking something and forgot to revert the change